docs/algorithms/kem/bike.yml

```diff
@@ -29,6 +29,7 @@ upstream-ancestors:
 - https://bikesuite.org/files/v4.1/Reference_Implementation.2020.10.20.2.zip
 parameter-sets:
 - name: BIKE-L1
+  oqs_alg: OQS_KEM_alg_bike_l1
```

Comment:
Wouldn't it be possible to generate these IDs automatically such as to avoid creating a manual "forward dependency" to oqs-openssl?

Reply:
Yes, I think so. But as mentioned in https://github.com/open-quantum-safe/liboqs/issues/1336#issuecomment-1361281296, I would at the moment remove these IDs and avoid "forward references" to openssl.

Reply:
removed forward references